Assets/Scripts/navigation/Navigation.cs

```diff
@@ -0,0 +1,186 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+using UnityEngine.UI;
+using TMPro;
+
+namespace nav {
+	public class Navigation : MonoBehaviour
+	{
+		public GameObject TitleScreen;
+		Transform player;
+		private city cities;
+		[SerializeField]
+		private NavMeshAgent AI;
+		[SerializeField]
+		//private LineRenderer lineRenderer;
+		public TMP_Text text;
+		public Image ImageSlot;
+		private const string ResourcePath = "crew_portraits";
+		private const string DefaultPortrait = "crew_portraits/phoenician_sailor";
+		public GameObject menu;
+		public GameObject Navgater;
+		private bool StartNavigation = false;
```

Comment:
we also aren't using PascalCase for private fields, use _startNavigation for those instead. you can see how we handle different cases here:
https://github.com/kddressel/sailingwiththegods/blob/develop/docs/coding-convention.md